bumble/utils.py

```diff
@@ -21,7 +21,7 @@
 import collections
 from functools import wraps
 from colors import color
-from pyee import EventEmitter
+from pyee import EventEmitter as BaseEventEmitter
```

Comment:
It might be safer to leave `EventEmitter` as-is, and name the subclass something else (maybe `AbortableEventEmitter`. This will be make less error prone for importers, as they will explicitly decide to import one of the other, not based on where the import comes from (from `pyee` directly or from `utils.py`).